test/azure/generated/AzureCompositeModelClient/operations/primitive.ts

```diff
@@ -2068,8 +2068,8 @@ export class Primitive {
     let field = (options && options.field !== undefined) ? options.field : undefined;
     // Validate
     try {
-      if (field && !Buffer.isBuffer(field)) {
-        throw new Error('field must be of type buffer.');
+      if (field && !(field instanceof Uint8Array)) {
+        throw new Error('field must be of type uint8array.');
```

Comment:
Should the casing be the same here as in the previous line (Uint8Array)?

Reply:
That seems best, but we probably have a .ToLower() in this generator step for some reason.